ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml

```diff
@@ -15,16 +15,23 @@
     dest: /home/iojs/get-pip.py
   when: pip_exists.stat.exists == False
 
-- name: install pip
+- name: install pip AIX71
   raw: /usr/bin/python /home/iojs/get-pip.py
-  when: pip_exists.stat.exists == False
+  when: pip_exists.stat.exists == False && os == "aix71"
+
+- name: install pip AIX72
+  raw: /opt/bin/python3 /home/iojs/get-pip.py
+  when: pip_exists.stat.exists == False && os == "aix72"
 
 - name: create symbolic link
   file:
-   src: "/opt/freeware/bin/pip"
+   src: "/opt/bin/pip"
    dest: "/usr/bin/pip"
    state: link
   when: pip_exists.stat.exists == False
 
 - name: install tap2junit
-  pip: name=tap2junit state=present
+  pip: 
+    name: tap2junit
+    state: present
+    executable: /opt/bin/pip
```

Comment:
Is this supposed to be `/usr/bin/pip`?

Reply:
nope supposed to be `opt/freeware/bin/pip` according to the commit history

Reply:
wait scratch that reading the wrong part of the code